node.ks.in

```diff
@@ -12,7 +12,7 @@ fi
 timezone --utc UTC
 lang en_US.UTF-8
 keyboard us
-auth --enableshadow --passalgo=sha512
+authselect minimal
```

Comment:
`authselect select minimal`

Reply:
or drop the whole thing? it apparently worked like this too...:)

Reply:
I don't think it worked, or at least there was completely different profile and only on RHVH

Reply:
can't fins ks logs, but i booted it and the profile is set correctly to minimal anyway, likely from the rhvh's own ks. So i'll drop the whole thing here then...

Reply:
works for rhvh, for node it likely needs https://github.com/oVirt/ovirt-node-ng-image/pull/10